lib/exhal/document.ex

```diff
@@ -4,6 +4,7 @@ defmodule ExHal.Document do
   """
 
   alias ExHal.Link, as: Link
+  alias ExHal.NsReg, as: NsReg
```

Comment:
You can leave off the `as: NsReg` for brevity.

[Notice that calling alias without the as: option automatically sets an alias based on the last part of the module](http://elixir-lang.org/docs/v1.2/elixir/Kernel.SpecialForms.html#alias/2)